schema/pgsql/upgrades/1.2.1.sql

```diff
@@ -8,6 +8,7 @@ COMMENT ON COLUMN notification.properties_checksum IS 'sha1(all properties)';
 ALTER TABLE timeperiod_range ALTER COLUMN range_value TYPE text;
 
 ALTER TABLE checkcommand_argument ALTER COLUMN argument_key TYPE varchar(255);
+ALTER TABLE checkcommand_argument ALTER COLUMN argument_key SET STORAGE PLAIN;
```

Comment:
What's that for?

Reply:
Please take a look at this comment <https://github.com/Icinga/icingadb/pull/871#issuecomment-2522929215>, especially the collided PostgreSQL detail block.

AFAIK, setting a storage type is mostly over-optimizing, but otherwise there was a diff between a plain schema and a schema upgrade.

Reply:
Changed as discussed and did the [schema comparison dance](https://github.com/Icinga/icingadb/pull/871#issuecomment-2522929215) again: no diff, looks good.

```diff
--- pr871-pgsql-after-upgrade.sql       2024-12-17 17:11:35.087116201 +0100
+++ pr871-pgsql-schema-import.sql       2024-12-17 17:11:52.071164045 +0100
@@ -4395,8 +4395,7 @@
 --

 COPY public.icingadb_schema (id, version, "timestamp") FROM stdin;
-1      3       1734451865380
-2      4       1734451889907
+1      4       1734451909295
 \.


@@ -4668,7 +4667,7 @@
 -- Name: icingadb_schema_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
 --

-SELECT pg_catalog.setval('public.icingadb_schema_id_seq', 2, true);
+SELECT pg_catalog.setval('public.icingadb_schema_id_seq', 1, true);


 --
```